pkg/v1/providers/tests/unit/tkr-bom-v1.21.1.yaml

```diff
@@ -0,0 +1,483 @@
+# taken from tkr-bom-v1.21.2+vmware.1-tkg.2-20210924-539f8b15.yaml 9/29/2021
```

Comment:
nit: I think this may be better placed in `pkg/v1/providers/tests/unit/fixtures`

also, would it be worth capturing something additional in the comments here and/or in the readme about how to update these bom files when desired or necessary?